languages/julia/config.json

```
@@ -9,6 +9,14 @@
     },
     "exercises": {
         "concept": [
+            {
+                "uuid": "39ebdd04-4f84-4817-bf9d-f1f9e066c283",
+                "slug": "lasagna",
```

This must match the exercise's directory name:

```suggestion
                "slug": "functions-introduction",
```